src/loader.cpp

```diff
@@ -36,7 +46,10 @@ loaded_morphology load_neuroml(const std::filesystem::path &fn) {
 }
 
 loaded_morphology load_asc(const std::filesystem::path &fn) {
-    loaded_morphology result;
+    auto m = arborio::load_asc(fn);
+    loaded_morphology result{.morph=m.morphology};
+    for (const auto &[k, v]: m.labels.regions()) result.regions.emplace_back(k, to_string(v));
```

Comment:
A tiny formatting nitpick: `auto& []` instead of `auto &[]`.

Reply:
Ha, serves me right for cut and pasting from your code above ;)

Reply:
So true.